dissect/target/plugins/filesystem/walkfs.py

```diff
@@ -57,6 +57,7 @@ def generate_record(target: Target, path: TargetPath) -> FilesystemRecord:
         atime=from_unix(stat.st_atime),
         mtime=from_unix(stat.st_mtime),
         ctime=from_unix(stat.st_ctime),
+        btime=from_unix(stat.st_birthtime),
```

Comment:
stat.st_birthtime could be `None` which makes `from_unix()` fail, so it should be guarded.

Reply:
Gah, ofc. I promise to run the tests next time.